pkg/kwokctl/vars/vars.go

```diff
@@ -171,11 +171,28 @@ var (
 		return ""
 	}()
 
+	// Mode is several default parameter templates for clusters
+	Mode = getEnv("KWOK_MODE", "")
```

Comment:
So the default mode would be "", is it intentional?

Reply:
Yes, No special configuration is done by default.